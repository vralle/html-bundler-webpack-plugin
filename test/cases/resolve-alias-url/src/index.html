<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <link href="@images/favicons/favicon.ico" rel="icon" />
  </head>
  <body>
    <h1>Hello World!</h1>
    <!-- resolve alias from local directory -->
    <img src="@images/image.webp">
    <!-- resolve alias as external url -->
    <img src="external/plugin-logo.png">
  </body>
</html>