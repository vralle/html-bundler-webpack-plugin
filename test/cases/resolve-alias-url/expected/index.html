<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <link href="img/favicon.edda23bf.ico" rel="icon" />
  </head>
  <body>
    <h1>Hello World!</h1>
    <!-- resolve alias from local directory -->
    <img src="img/image.d4711676.webp">
    <!-- resolve alias as external url -->
    <img src="https://raw.githubusercontent.com/webdiscus/html-bundler-webpack-plugin/refs/heads/master/images/plugin-logo.png">
  </body>
</html>